### PR TITLE
Make update for topics

### PR DESCRIPTION
**Pull request by OrestF**
As an admin user I want to be able to rename a topic

### CODE REVIEW

File: app/controllers/topics_controller.rb [hunk 1]
```diff
@@ -21,12 +22,24 @@ def create
       end
   end
 
-  private
+	def edit
```
Comment (ruba-ruba) [inline app/controllers/topics_controller.rb]: indentation

File: app/controllers/topics_controller.rb [hunk 2]
```diff
@@ -21,12 +22,24 @@ def create
...
+    @topic = Topic.find(params[:id])
+  end
+
+	def update
```
Comment (ruba-ruba) [inline app/controllers/topics_controller.rb]: indentation

File: app/controllers/topics_controller.rb [hunk 3]
```diff
@@ -21,12 +22,24 @@ def create
...
+    @topic = Topic.find(params[:id])
+    if @topic.update_attributes(topic_params)
+      flash[:success] = 'Topic updated'
+      redirect_to @topic
+    else
+      render 'edit'
+    end
+  end
+
+	private
```
Comment (ruba-ruba) [inline app/controllers/topics_controller.rb]: see https://github.com/bbatsov/ruby-style-guide
and find how to align private/protected methods

**Comment by OrestF** [comment]
Fix indents

**Pull request merged**